tcl/org.testeditor.tcl.dsl.ide.tests/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding/<project>=UTF-8
```

Comment:
hey, that is eclipse, I did not question that.
however, I included some of the eclipse specific stuff so that upon checkout eclipse works with the project (out of the box)